external_auth/views.py

```diff
@@ -49,13 +53,19 @@ def pakwheels_registration(request):
             user.profile.city = pakwheels_profile.city
             user.profile.country = pakwheels_profile.country
             user.save()
-            messages.success(
-                request,
-                'Pakwheels login successful.'
-                ' We have registered you on our website with same credentials.'
-                ' You can change your password.',
+            authenticated = authenticate(
```

Comment:
Same here no need to authenticate. just do forceful login and set the user active.

If a user registers on `auto-shop` directly then login the user but the user wont be active unless we make the user active (We will add a feature later to allow user to activate the account by sending email with activation link).
If a user registers with `Pakwheels` then not only login the user but also make it active because the user comes from an authenticated source.